pkgs/development/python-modules/md2gemini/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, mistune_2, cjkwrap, wcwidth, pytest }:
+
+buildPythonPackage rec {
+  pname = "md2gemini";
+  version = "1.8.1";
+
+  propagatedBuildInputs = [ mistune_2 cjkwrap wcwidth ];
+  checkInputs = [ pytest ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0mfa0f0m762168fbsxjr1cx9yhj82dr8z1d28jl6hj9bkqnvvwiy";
+  };
+
```

```suggestion

  propagatedBuildInputs = [ mistune_2 cjkwrap wcwidth ];
  checkInputs = [ pytestCheckHook ];
  pythonImportsCheck = [ "md2gemini" ];

```

pytest alone does not run pytest.